README.md

```diff
@@ -1,21 +1,17 @@
-<h1 align="center">Hi 👋, I'm Ashish</h1>
+<h1 align="center">Hi, I'm Ashish 👋</h1>
 <h3 align="center">A passionate FullStack Developer from India</h3>
 
-<p align="left"> <img src="https://komarev.com/ghpvc/?username=ashish-devv&label=Profile%20views&color=0e75b6&style=flat" alt="ashish-devv" /> </p>
-
 <hr>
 
+- 🌱 I’m currently learning **Data Science and Android development**
 
+- 💬 Ask me about **Flask, AngularJs, Spring, Node, ...**
 
-- 🌱 I’m currently learning **Data Science And Android development**
-
-- 💬 Ask me about **Flask,AngularJs,Spring,Node...**
-
-- 📫 How to reach me **akashish908@gmail.com**
+- 📫 How to reach me: **akashish908@gmail.com**
 
-- ⚡ Fun fact **I love myself more !..**
+- ⚡ Fun fact: **I love myself more !**
```

Comment:
```suggestion
- ⚡ Fun fact: **I love myself more!**
```

Reply:
I don't think that's valid. There should be no spaces before the `!`.